### PR TITLE
Add an app setting when creating a custom container web app on linux …

### LINKED ISSUE #4367: Add an app setting when creating a custom container web app on linux
### Description
Add an app setting when creating a custom container web app on linux

---

### Environment summary

**Install Method:** How did you install the CLI? (e.g. pip, interactive script, apt-get, Docker, MSI, nightly)  
Answer here:

**CLI Version:** What version of the CLI and modules are installed? (Use `az --version`)  
Answer here:

**OS Version:** What OS and version are you using?  
Answer here:

**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, Bash on Windows)  
Answer here:

### PR DESCRIPTION
**Pull request by panchagnula**
Closes #4367

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -82,6 +84,10 @@ def create_webapp(resource_group_name, name, plan, runtime=None, startup_file=No
                              deployment_source_branch, deployment_local_git)
 
     _fill_ftp_publishing_url(webapp, resource_group_name, name)
+
+    if linux_storage_default:
+        update_app_settings(resource_group_name, name, ["WEBSITES_ENABLE_APP_SERVICE_STORAGE=false"])
```
Comment (LukaszStem) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: We can reduce these 4 lines to 1. Just bring `update_app_settings(resource_group_name, name, ["WEBSITES_ENABLE_APP_SERVICE_STORAGE=false"])` to the affected `elif` branch

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py [hunk 2]
```diff
@@ -82,6 +84,10 @@ def create_webapp(resource_group_name, name, plan, runtime=None, startup_file=No
                              deployment_source_branch, deployment_local_git)
 
     _fill_ftp_publishing_url(webapp, resource_group_name, name)
+
+    if linux_storage_default:
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: Can you set this as part of `client.web_apps.create_or_update`?
Also please update test cases to incorporate this change, such as the `test_linux_webapp_quick_create`

Comment (LukaszStem) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: We can have just 1 line ->             `site_config.app_settings.append(NameValuePair('WEBSITES_ENABLE_APP_SERVICE_STORAGE', "false"))` in the `elif` branch

Comment (panchagnula) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: the setting can be applied only after the app is created and that was the reason for not having this inside the elif block

**Pull request merged**